message.go

```diff
@@ -12,22 +12,26 @@ type Message struct {
 	TimeToLive            int                    `json:"time_to_live,omitempty"`
 	RestrictedPackageName string                 `json:"restricted_package_name,omitempty"`
 	DryRun                bool                   `json:"dry_run,omitempty"`
-	Notification          *Notification           `json:"notification"`
+	Notification          Notification           `json:"notification"`
 }
 
 type Notification struct {
 	Title string `json:"title"`
 	Body  string `json:"body"`
 }
 
+func (notification *Notification) isValid() bool {
+	return notification.Title != "" && notification.Body != ""
+}
```

Comment:
just thinking.. could it be the case where there is no title for example and only body? Or it will not work anyway on the new Android?

Reply:
good question! Actually having no idea now as everything in payload being documented as optional haha.
Currently, if it couldn't be a case as we fill all in `android.go`

Reply:
Why do we want to validate it if it is optional on android side? Does it protect us from not sending notifications silensely?

Reply:
It helps us to display the notification on user device. Why do we want to send it silently?

Reply:
ah yes, it protect us from not sending notification silently 😄 
I would say that only helps our case currently

Reply:
But also thinking to remove it and make this one as pure as oriignal API from FCM ... 🤔

Reply:
agreed with the last point

Reply:
done